### PR TITLE
Add tweak to make reblog peepr target posts

### PR DESCRIPTION
**Pull request by akunohomu**
This makes the peepr of userB (in the case of `userA🔁userB`) target their specific post/reblog, instead of their blog in general.

### CODE REVIEW

**Comment by akunohomu** [comment]
The tweak summary is confusing, I don't know how to write better. To explain, when you see
`userA 🔁 userB`
on your dash, and click `userB`, right now Tumblr will open the sidebar to userB's blog. This tweak makes the sidebar open to *userB's specific post*.

Rationale for being default: sidebar has post IDs everywhere already, it's just the dashboard/likes that's missing post IDs. Even drafts/queue work like this.

This doesn't bother to make `userA` target userA's specific post. If you're on the dashboard looking at a post there is no reason to open the sidebar to it.

Issues you may run into:
* If the post was deleted, you will probably be unable to view any of the user's posts in spawned sidebar, only able to view blog title/description. This is a Tumblr problem and already happens when using sidebar and when clicking Source links.
* If you switch this tweak on or off and, without reloading, click a peepr *you have already clicked* the behavior will not change; it'll still open to the post/blog as before. I have no idea why it does this and assume it's Tumblr caching it in the code somewhere; the destroy code definitely removes postId.

File: Extensions/tweaks.js [hunk 1]
```diff
@@ -797,6 +806,36 @@ XKit.extensions.tweaks = new Object({
 
 	},
 
+	add_id_to_reblog_peepr: function() {
+		"use strict";
+		var where = XKit.interface.where();
+		if (!where.dashboard && !where.likes)
+			return;
```
Comment (nightpool) [inline Extensions/tweaks.js]: add braces. (not sure why eslint didn't catch this)

**Comment by hobinjk** [comment]
It looks like this is now the default Tumblr behavior which is both nice and a sign that we need to be a tad faster at reviewing pull requests.

**Comment by akunohomu** [comment]
I was definitely seeing that behavior yesterday too but no longer am. Hopefully they turn it back on and I don't need to fix my code

**Comment by AprilSylph** [comment]
this is now default tumblr behaviour on the new dashboard

**Pull request closed without merging**